client/normal_new.go

```diff
@@ -348,11 +351,52 @@ func convertAnyArrToNillableIntOrFloatArr(arr []any) (NormalValue, error) {
 	return NewNormalNillableIntArray(result), nil
 }
 
-func convertAnyArrToFloatArr(arr []any) (NormalValue, error) {
+func convertAnyArrToFloat32Arr(arr []any) (NormalValue, error) {
+	result := make([]float32, len(arr))
+	for i := range arr {
+		if arr[i] == nil {
+			return convertAnyArrToNillableFloat32Arr(arr)
+		}
+
+		var floatVal float32
+		switch v := arr[i].(type) {
+		case float32:
+			floatVal = v
+		case float64:
+			return convertAnyArrToFloat64Arr(arr)
```

Comment:
question: The whole array will be converted to Float64?

Reply:
Yes. There was a similar pattern before for the float64 function. It's the same thing here.